packages/next/server/dev/next-dev-server.ts

```diff
@@ -296,13 +294,25 @@ export default class DevServer extends Server {
 
       files.push(...envFiles)
 
+      console.log('directories', directories)
```

Comment:
Unnecessary console.log here?

Reply:
Yeah good catch, looks like that didn't get pushed